rule.php

```diff
@@ -155,6 +170,10 @@ public static function validate_settings_form_fields(array $errors,
         $quizsettings = (new quiz_settings())->from_record($settings);
         // Set non-form fields.
         $quizsettings->set('quizid', $data['instance']);
+        // Explicit setting the sebconfigfile value such that the validation will check for a valid plist.
+        if (isset($data['filemanager_sebconfigfile'])) {
+            $quizsettings->set('sebconfigfile', $data['filemanager_sebconfigfile']);
```

Comment:
in DB table quizaccess_seb_quizsettings "sebconfigfile' and persistent class is text. Do you think we need to change it?

Reply:
This was my hack around to keep that value in the persistent class, and it will store the itemid to acknowledge that the file is been uploaded. In conjunction with the workarounds I mentioned above.